res/values/strings.xml

```diff
@@ -215,6 +221,12 @@
     <string name="menu_zoom_in">Zoom in</string>
     <string name="menu_zoom_out">Zoom out</string>
     <string name="menu_save_log">Save log</string>
+    <string name="menu_more_options">More Options</string>
+    <string name="menu_learn_spelling">Learn Spelling</string>
+    <string name="menu_chat_audit_log">Chat Audit Log</string>
+    <string name="jump_to_message">Jump to Message<string>
+    <string name="copy_json">Copy JSON</string>
+    <string name="replace_draft">replace draft</string>
```

Comment:
should that really start lower case? if so, add a hint for the translators.

Reply:
I use it for a button that does the all uppercase transformation so I did not care.
```suggestion
    <string name="replace_draft">Replace draft</string>
```